src/rgw/support/rgw_support_bundle

```diff
@@ -78,6 +79,41 @@ class RGWSupportBundle:
                     outfile = os.path.join(RGWSupportBundle._tmp_src, file)
                     if os.path.isfile(infile):
                         shutil.copyfile(infile, outfile)
+
+        if duration:
+            # files in delim_space use timestamp syntax as
+            # [2022-07-10 05:26:57,579] & 2022-07-10 05:26:34
+            # files in delim_T use timestamp syntax as
+            # 2022-07-10T05:28:35.570+0000
+            from cortx.rgw.const import ( RGW_STARTUP_LOG, RGW_SETUP_LOG,
+                RGW_SUPPORT_BUNDLE_LOG, RADOSGW_ADMIN_LOG, RGW_1_LOG )
+
+            # log files with timestamp syntax "YYYY-mm-DD H:M:S"
+            delim_space = [RGW_STARTUP_LOG, RGW_SETUP_LOG, RGW_SUPPORT_BUNDLE_LOG]
+            # log files with timestamp syntax "YYYY-mm-DDTH:M:S"
+            delim_T = [RADOSGW_ADMIN_LOG, RGW_1_LOG]
+            date_regex = '[0-9]{4}-[0-9]{2}-[0-9]{2}' # regex eg "1995-09-27"
```

Comment:
@rohit-k-dwivedi  could you pls move this two regex to const.py file ?
in future it can be reused in our code.

Reply:
done